src/useAspect.tsx

```diff
@@ -1,6 +1,6 @@
 import { useThree } from 'react-three-fiber'
 
-export function useAspect(type: string, width: number, height: number, factor: number = 1) {
+export function useAspect(type: 'cover', width: number, height: number, factor: number = 1): [number, number, number] {
```

Comment:
if there is only one option for the `type`, should not it be omitted, and used as value?

Reply:
I guess it's a future-proofing for an api, because it should support other css-like modes, though nobody wrote that support yet.